lib/nxdk/format.c

```diff
@@ -84,7 +84,7 @@ bool nxFormatVolume (const char *volumePath, uint32_t bytesPerCluster)
         return false;
     }
 
-    RtlFillMemory(buffer, alignment, 0xFF);
+    memset(buffer, 0xFF, alignment);
```

Comment:
nit: One of the few cases where `RtlFillMemory` and friends might be useful, because this is essentially a kernel driver.
When working on a kernel driver, one might not have access to a full libc and the kernel is definitely available, too (so there's no risk of adding an unnecessary dependency).

However, at the moment this also uses `malloc` and friends, so it doesn't really matter.
For now, I prefer `memset`, but I wouldn't be surprised if we change it back eventually (when turning this into a low-level driver).